web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -137,8 +137,14 @@ function isR2L() {
             settings.presets = [];
             settings.presets.push({
                 name: settings.allText,
-                all: true
             });
+
+            if ($select.attr('id') === 'statuses') {
```

Comment:
This works fine, but I wonder, for possible futureproofing, if we could use this path if `$select.data('all-options')` exists? Currently that would only catch the same element, but I think it makes sense to use all-options if present in case it's added to something else.